GoogleForCreators/web-stories-wp
# Add storybook story for failed preview dialog

## Summary

Adds a new storybook story to make testing this component easier. Difficult to trigger it otherwise.

See for context https://github.com/google/web-stories-wp/pull/1656/files#r430941190

## User-facing changes

None in the  editor. Storybook only.

## Testing Instructions

1. `npm run storybook`
1. Open story
1. See result:

<img width="578" alt="Screenshot 2020-05-28 at 15 04 14" src="https://user-images.githubusercontent.com/841956/83145456-1cde8c00-a0f5-11ea-8efb-a505fea22eb9.png">

---



See #1656
